decidim-ai/lib/decidim/ai/spam_detection/spam_detection.rb

```diff
@@ -28,6 +28,12 @@ module Strategy
         autoload :Bayes, "decidim/ai/spam_detection/strategy/bayes"
       end
 
+      # When the engine is consistently marking spam content without errors,
+      # you can skip human intervention by enabling this functionality
+      config_accessor :hide_reported_resources do
```

Comment:
It's important to clarify always when it is a manual or an automatic action, because if not it's difficult as an implementer to know what exactly means `hide_reported_resources` IMO

```suggestion
      config_accessor :hide_reported_resources_automatically do
```

Reply:
done in 5080ffcefe